po/uk.po

```diff
@@ -539,13 +539,15 @@ msgstr "Приховати нестабільні та RC випуски"
 
 #: src/Gtk/SettingsDialog.vala:113
 msgid "Hide failed or incomplete builds"
-msgstr ""
+msgstr "Приховати невдалі або незавершені збірки"
 
 #: src/Gtk/SettingsDialog.vala:114
 msgid ""
 "If a kernel version exists on the mainline-ppa site, but is an incomplete or "
 "failed build for your arch ("
 msgstr ""
+"Якщо версія ядра існує на сайті mainline-ppa, але є неповною або "
+"невдала збірка для вашої архи ("
```

Comment:
string got broke in half, so just now I updated it to use printf and %s so it makes more sense for translating, so this patch is obsolete now, see the current version instead